src/System.Net.NetworkInformation/tests/FunctionalTests/IPInterfacePropertiesTest.cs

```diff
@@ -19,6 +19,7 @@ public IPInterfacePropertiesTest()
         }
 
         [Fact]
+        [PlatformSpecific(PlatformID.Windows)] // Linux and OSX do not support some of these
```

Comment:
I'll be adding more unix-specific tests separately, and I will probably split these into more granular tests then.